### DISCUSSION TITLE
OTF Fonts. Can they be added?

### DISCUSSION THREAD

**Discussion by onjaysantana** [givanz/Vvveb#121] [Q&A]
I use a few custom OTF fonts but can't figure out how to set them up in the editor. I've tried using CSS to define the fontface and assign page elements to classes, but it is not working.  

I expect I'm missing something really fundamental and simple. Any help would be so welcome! 

Thanks,
JS

**Comment by givanz** [comment]
Thanks for the suggestion.

I'm preparing a plugin to upload custom fonts.

The fonts will show in the page builder inline text editor font drop down and in styles for font family option.

I will get back when I finish the plugin.

**Comment by onjaysantana** [comment]
Hello,

Thanks very much for taking the time to reply. I look forward to your
updates. Great work!

Best,
JS
